content/blog/experiments-llm/index.Rmd

```diff
@@ -0,0 +1,170 @@
+---
+output: hugodown::hugo_document
+
+slug: experiments-llm
+title: Three experiments in LLM code assist with RStudio and Positron
+date: 2025-01-29
+author: Simon Couch
+description: >
+    We've been experimenting with LLM-powered tools to streamline R
+    data science and package development.
+    
+photo:
+  url: https://unsplash.com/photos/AgxNjvE8KTE
+  author: Dibya Jyoti Ghosh
+
+categories: [roundup] 
+tags: [ellmer, ai]
```

Comment:
Both of these are the first instance of the tag, though maybe could have some shared tags with #719.